packages/react-components/src/organisms/__tests__/EventConversation.test.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { ComponentProps } from 'react';
+import { render } from '@testing-library/react';
+import { createGroupResponse } from '@asap-hub/fixtures';
+
+import EventConversation from '../EventConversation';
+
+const props: ComponentProps<typeof EventConversation> = {
+  ...createGroupResponse(),
+};
+
+it('renders when group with slack tool is provided', () => {
+  const { queryByRole, getByRole, rerender } = render(
+    <EventConversation {...props} tools={{}} />,
+  );
+  expect(queryByRole('heading')).toBeNull();
+  rerender(
+    <EventConversation {...props} tools={{ slack: 'http://example.com' }} />,
+  );
+  expect(getByRole('heading').textContent).toMatchInlineSnapshot(
+    `"Continue the conversation"`,
+  );
+});
+
+it('renders slack tool button', () => {
```

Comment:
slightly strange you test one but not the other, but not gonna complain since they don't seem so essential to test :D